RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -783,7 +810,7 @@ boolean autoConsume(ConsumeAction action)
 		abort("ConsumeAction not prepped: " + to_debug_string(action));
 	}
 
-	if (action.organ == AUTO_ORGAN_LIVER)
+	if (action.organ == AUTO_ORGAN_LIVER && action.it != $item[tiny stillsuit])
```

Comment:
excluding from canOde(item toDrink) is probably needed too for Drunk and Avuncular

Reply:
Done